holidays/calendars/chinese.py

```diff
@@ -16,6 +16,10 @@
 from holidays.calendars.custom import _CustomCalendar
 from holidays.calendars.gregorian import JAN, FEB, MAR, APR, MAY, JUN, SEP, OCT, NOV
 
+CHINESE_CALENDAR = "CHINESE_CALENDAR"
+KOREAN_CALENDAR = "KOREAN_CALENDAR"
+VIETNAM_CALENDAR = "VIETNAM_CALENDAR"
```

Comment:
This might be better for uniformity https://en.m.wikipedia.org/wiki/Vietnamese_calendar
```suggestion
VIETNAMESE_CALENDAR = "VIETNAMESE_CALENDAR"
```